includes/sanitizers/class-amp-layout-sanitizer.php

```diff
@@ -20,8 +20,11 @@ class AMP_Layout_Sanitizer extends AMP_Base_Sanitizer {
 	public function sanitize() {
 		$xpath = new DOMXPath( $this->dom );
 
-		// Elements with the `layout` attribute will be validated by `AMP_Tag_And_Attribute_Sanitizer`.
-		$nodes = $xpath->query( '//*[ not( @layout ) and ( @data-amp-layout or @width or @height or @style ) ]' );
+		/**
```

Comment:
Just noticed this, that it should have been `/*` instead of `/**` since it is not a phpDoc comment. Not a big deal but we can fix it as part of another ongoing PR.